pages/report.tsx

```diff
@@ -127,6 +158,15 @@ export default function Report() {
                     <SharkWasReleasedCheckboxField name="was_released" />
                   </Box>
 
+                  <Input
+                    multiple
+                    mb={2}
+                    pt={1}
+                    type="file"
+                    onChange={onFileChange}
+                    accept="image/*"
+                  />
```

Comment:
A form label and hint would be very helpful here: we should tell users specifically that this field accepts up to N images under 5MB.